src/nunjucks/moduk/components/phase-banner/macro.njk

```diff
@@ -0,0 +1,10 @@
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+{% macro modukPhaseBanner(params) %}
+  {{ govukPhaseBanner({
+    tag: params.tag | addCustomMODUKClass("moduk-tag--default", { not: r/^govuk-tag--/ }),
+    html: params.html,
+    text: params.text
+  }) }}
+{% endmacro %}
+
```

Comment:
I notice that the govuk frontend repo there had a tag property that accepts {text, classes} so instead of applying css and overriding from a parent selector I reused the changes from the tag component. The downside is I had to explicitly assign all the variables otherwise I would have to create a custom filter.

Reply:
Hmm there are also `classes` and `attributes` properties:

![image](https://user-images.githubusercontent.com/66470099/202753932-c942aef1-62f8-453c-a900-1d0fa8fcdc35.png)

Another possible option could be to add a `path` option to `addCustomMODUKClass`, that takes something that works with lodash [`get()`](https://lodash.com/docs/4.17.15#get) and [`set()`](https://lodash.com/docs/4.17.15#set) (e.g. `"tag.classes"`).

But don't mind, this works.